api/v1beta1/aerospikecluster_types.go

```diff
@@ -848,7 +848,8 @@ type AerospikePodStatus struct { //nolint:govet // for readability
 // +kubebuilder:subresource:status
 
 // AerospikeCluster is the schema for the AerospikeCluster API
-// +operator-sdk:csv:customresourcedefinitions:displayName="Aerospike Cluster",resources={{Service, v1},{Pod,v1},{StatefulSet,v1}}
+//
+//nolint:lll // for readability  // +operator-sdk:csv:customresourcedefinitions:displayName="Aerospike Cluster",resources={{Service, v1},{Pod,v1},{StatefulSet,v1}}
```

Comment:
IMO, we should follow one standard for adding nolint marker. In the beginning or at the end or above the line